docs/data/joy/getting-started/templates/email/components/WriteEmail.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+import Box from '@mui/joy/Box';
+import ModalClose from '@mui/joy/ModalClose';
+import Button from '@mui/joy/Button';
+import FormControl from '@mui/joy/FormControl';
+import Textarea from '@mui/joy/Textarea';
+import Sheet from '@mui/joy/Sheet';
+import { IconButton, Input, Stack, Typography } from '@mui/joy';
+
+import FormatColorTextRoundedIcon from '@mui/icons-material/FormatColorTextRounded';
+import AttachFileRoundedIcon from '@mui/icons-material/AttachFileRounded';
+import InsertPhotoRoundedIcon from '@mui/icons-material/InsertPhotoRounded';
+import FormatListBulletedRoundedIcon from '@mui/icons-material/FormatListBulletedRounded';
+
+interface WriteEmailProps {
+  open?: boolean;
+  onClose?: () => void;
+}
+
+const WriteEmail = React.forwardRef<HTMLDivElement, WriteEmailProps>(
+  function DrawerMobileNavigation({ open, onClose }, ref) {
+    return (
+      <Sheet
+        ref={ref}
+        sx={{
+          alignItems: 'center',
+          px: 1.5,
+          py: 1.5,
+          ml: 'auto',
+          width: { xs: '100dvw', md: 600 },
+          flexGrow: 1,
+          border: '1px solid',
+          borderRadius: 'md',
+          backgroundColor: 'var(--joy-palette-background-level1)',
+          borderColor: 'var(--joy-palette-neutral-outlinedBorder)',
+          boxShadow: 'lg',
+          zIndex: 1000,
+          position: 'fixed',
+          bottom: 0,
+          right: 24,
+          transform: open ? 'translateY(0)' : 'translateY(100%)',
+          transition: 'transform 0.3s ease',
+        }}
+      >
+        <Box sx={{ mb: 2 }}>
+          <Typography level="title-sm">New message</Typography>
+          <ModalClose id="close-icon" onClick={onClose} />
+        </Box>
+
+        <Box
+          sx={{ display: 'flex', flexDirection: 'column', gap: 2, flexShrink: 0 }}
+        >
+          <FormControl>
+            <Typography level="title-sm">To</Typography>
```

Comment:
```suggestion
            <FormLabel>To</FormLabel>
```
FormLabel is better in term of accessibility.